website/client/src/components/groups/group.vue

```diff
@@ -486,6 +486,12 @@ export default {
         this.$router.push('/');
       }
     },
+    async group () {
```

Comment:
instead of watching the entire group object could we watch just the id? So it's only executed when the group actually changes `async 'group._id'`